spec/rubocop/cop/lint/number_conversion_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Lint::NumberConversion do
+  subject(:cop) { described_class.new(config) }
+
+  let(:config) { RuboCop::Config.new }
+
+  it 'registers an offense when using `#to_i`' do
+    expect_offense(<<-RUBY.strip_indent)
+      '10'.to_i
+      ^^^^^^^^^ Replace unsafe number conversion with number class parsing, instead of using 10.to_i, use stricter Integer("10").
```

Comment:
The actual receiver instance is a String object (Not Integer object) . It's better to show `instead of using '10'.to_i` instead of `instead of using 10.to_i`. It will help users understand :-)

Reply:
Just noticed it, fixed :)